\documentclass[12pt,letterpaper]{article}
%\documentclass{article}
\raggedright
\usepackage{natbib}
\usepackage{times}
\usepackage[T1]{fontenc}
\usepackage[pdftex]{graphicx}
\usepackage{sidecap}
\usepackage{amsmath}
\usepackage{multirow}
\usepackage{setspace}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{sectsty}
\usepackage{color}
\usepackage{geometry}
\usepackage{fullpage} % 1" margins

%\doublespacing
%\linespread{2.0}
\raggedright
\bibpunct{(}{)}{,}{a}{}{;}

\setlength{\parindent}{0.5in}
\setcounter{secnumdepth}{0}
\pagestyle{empty}

\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}

\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}

\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}

% scientific notation command
\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}

\begin{document}
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip

\medskip
\begin{center}

\noindent{\Large \bf Species interactions promote speciation in an adaptive 
radiation of flowering plants}
\bigskip

\noindent RH: CHARACTER DISPLACEMENT ACROSS A COMMUNITY MOSAIC 

\bigskip

\providecommand{\e}[1]{\ensuremath{\times 10^{#1}}}

\noindent {\normalsize \sc Deren~A.~R.~Eaton$^{1,*}$, Huang, S-Q.$^{2}$, and Richard~H.~Ree$^{3,4}$}\\
\noindent {\small \it 
$^1$Department of Ecology, Evolution, and Environmental Biology, Columbia University, New York, NY, 10027, USA;\\
%$^2$Chinese University...;\\
%$^3$Committee on Evolutionary Biology, University of Chicago, Chicago, IL, 60637, USA;\\
$^4$Botany Department, Field Museum of Natural History, Chicago, IL, 60605, USA}\\
\end{center}
$^\noindent{\bf Corresponding author:} Deren Eaton, de2356@columbia.edu.\\
\medskip


\vspace{0.5in}

\noindent Adaptive radiations are defined by the rapid diversification of species to fill a diversity of ecological niches. While the importance of ecological 
adaptations has been well studied in the context of diverse communities or clades, reproductive adaptations, which facilitate and maintain the isolation necessary for ecological divergences between species, are rarely studied at the same scale. Here we measure the strength and direction of selection on floral morphology imposed by the local floral morphospace within diverse communities of sympatric \emph{Pedicularis} species
in the Hengduan Mountains region of China. We show that variation across communities in the composition of species interactions drives divergent local adaptations within-species (between populations) that reduce gene flow and increase reproductive isolation. We employ a novel Bayesian implementation of a migration matrix-based approach to separating the effects of gene flow from selection in detecting local adaptation. Experimental crosses between divergent locally adapted populations of \emph{P. cranolopha} show that intraspecific floral differences affect assortative mating, consistent with genomic estimates for the direction and strength of gene flow. Together, these results suggest that interspecific interactions drive both interspecific and intraspecific reproductive character displacement, providing a link between species interactions and diversification within an adaptive radiation.  \\

% Adaptive radiation is defined by a rapid diversification of species to 
% fill a diversity of ecological niches.  \citep{Shluter}. 
% While the importance of ecological adaptations 
% has been well studied in the context of 
% diverse clades or communities 
% \citep{Losos, Cavender-Bares, Cichlid},
% %where multiple species may fill similar niches and thus impose parallel 
% %selection pressures, 
% reproductive adaptations, which facilitate and maintain the 
% isolation necessary for ecological divergences between species, are rarely 
% studied at the same scale \citep{groning_2008}.
% Here we measure the strength and direction of selection on 
% floral morphology imposed by the local floral morphospace
% within diverse communities of sympatric \emph{Pedicularis} species
% in the Hengduan Mountains region of China. 
% We show that variation across communities in the composition of species interactions 
% drives divergent local adaptations within-species (between populations)
% that reduce gene flow and increase reproductive isolation. 
% We employ a novel Bayesian implementation of a migration matrix-based 
% approach \citep{felsenstein_2002} to separating the effects of gene flow 
% from selection in detecting local adaptation. Experimental crosses between 
% divergent locally adapted populations of \emph{P. cranolopha} 
% show that intraspecific floral differences affect assortative mating, 
% consistent with genomic estimates for the direction and strength of gene flow. 
% Together, these results suggest that interspecific interactions drive 
% both interspecific and intraspecific reproductive character displacement, 
% providing a link between species interactions and diversification
% within an adaptive radiation.  \\

\vspace{1in}

\noindent (Keywords: Restriction-site associated DNA, 
migration, community structure, population, \emph{Pedicularis}, 
reproductive character displacement, reproductive interference)\\

\vspace{1.5in}

%\section*{Introduction}

\subsection*{Biotic interactions are important, reproductive especially in hotspots of diversity.}

The importance of biotic interactions in explaining global patterns of biodiversity has a long history in ecological theory \citep{schemske_2009}. The strength, frequency, and composition of species interactions experienced by individuals within a species can vary considerably, resulting in a mosaic of selection pressures \citep{thompson}, corresponding to differences in competition, mutualism, parasitism, and facilitation. 

--  Reproductive traits allow isolation...
--  Adaptive radiations compose many close relative diverging 
    concentrated in space. 
--  Reproductive character displacement
    and the divergence and isolation of populations provides a 
    link between species interactions and the process of speciation.


% Pedicularis in the Hengduan mountains, super diverse and vairable assemblages
The angiosperm genus \emph{Pedicularis} (Orobanchaceae)
consists of approximately 700 species of hemiparasitic herbs 
with a center of diversity in the Hengduan Mountains of south-central China
\citep{yang_flora_1998}. \emph{Pedicularis} exhibits spectacular 
interspecific variation in floral traits that is thought to 
reflect adaptations to reduce 
heterospecific pollen flow by generalist bumble bees
\citep{macior_pollination_1983, grant_modes_1994}. 
\cite{eaton_floral_2012} showed that species sort across the 
landscape into communities that minimize floral similarity. 


% Measuring character displacement is hard for species-rich clades, who do you compare? There is need for a 'community model of CD'

Eaton 2012 showed evidence of reproductive interference at the community level in Pedicularis. There is both a phentypic and phylogenetic signal, the latter perhaps a true expression of genetic distance playing a role in species interactions. 

To detect whether \emph{in situ} evolution is also occurring 
through local reproductive character displacement we measured
local floral phenotypes within 15 populations of a single 
widespread and variable species, \emph{P. cranolopha}
(Fig.~\ref{fig:1a-b}), and tested their with the local biotic 
environment composed by the floral morphospace occupied by 
synchronously flowering sympatric species of \emph{Pedicularis}. 

These tests require controlling for non-independence of populations or species as data points, which we do here. 


% SUPPLEMENT DETAILS ON THE METHOD
Attempts to overcome the phylogenetic challenges posed by both
incomplete lineage sorting and horizontal gene transfer are
represented by much recent work on multi-locus methods for inferring
species trees (or networks) based on gene tree coalescence
\citep[e.g.,][]{ane_bayesian_2007, liu_best:_2008, kubatko_stem:_2009,
kubatko_identifying_2009, yu_coalescent_2011}. These methods rely on
the availability of multiple unlinked nuclear markers with levels of
sequence variation appropriate for the phylogenetic scale of
inquiry. For recent divergence events in non-model organisms,
obtaining such multigene data sets remains challenging, because even
if a sufficient number of orthologous nuclear loci can be identified
and amplified reliably, individual genes are less likely over shorter
time scales to accumulate variation sufficient for resolving
informative gene trees \citep[e.g.,][]{omeara2010}.


% 
\section{Discussion}
Adaptive radiations can be defined by three core aspects: 
a diversity of ecological niches [], a rapid rate of diversification [], 
and a concentration of species in space []. In predicting the trajectory
of adaptive radiation significant focus is placed on existence of, and 
later saturation of, ecological opportunity, with the accumulation 
of species representing a negative density dependent process. Early in a 
radiation, however, species interactions may promote diversification[], for 
example through RCD[].  
 

%% Adaptive radiation occurs through a trajectory involving ecological
%% opportunity \citep{rainey}, divergence and isolation, and finally saturation. 


%% Within a widespread a variable species, locally adapted floral 
%% phenotypes with drastically different pistil lengths 

%% affect the ability for pollen tube growth, and thus
%% successful fertilizatoin between pops


%% Within a widespread and variable species, experimental crosses between
%% extremes show 

%% Parallel adaptation versus ancestry or gene flow...
%% Pollen - Pistil evolution...

%% Together these results 

%% A novel method




%% The extent to which species interactions influence rates of adaptation 
%% and diversification is a central question in ecology and evolution [2]. Reproductive 
%% character displacement (RCD) provides a mechanistic link between 
%% divergent selection imposed by interspecific interactions, 
%% and adaptations affecting assortative mating, and thus speciation. 
%% Here we show that variation in the stength and direction of selection
%% imposed by species interactions across a mosaic of 


%% in an adaptive radiation of \emph{Pedicularis}
%% in a hotspot of alpine plant diversity
%% that floral a widespread species 



%% RCD is well documented in the context of species pairs studied in sympatry versus allopatry, but 
%% less so in a community context, despite its posited role as a force accelerating 
%% diversification in adaptive radiations, where a concentration of closely related organismsm 
%% in space can intensify selection for assortative mating that reduces reproductive interference
%% or hybridization. 

   
%% Variation in the composition of local communities and
%% of interspecific interactions across an individual species 
%% geographic range can cause populations to
%% experience divergent selection [1,2,3], 
%% driving speciation. 
%% Character displacment may promote morphological 

%% Character displacement is proposed to drive the 

%% Adaptive radiations compose great numbers of 

%% While few studies have shown character displacement in the context
%% of sympatry versus allopatry of 

%% Variation in the strength or composition of interspecific 
%% interactions across a species geographic range can cause populations 
%% to experience divergent selection towards different local
%% phenotypic optima. When If such displacement involves repr

%% Sexual reproduction takes place within natural communities
%% where interactions bewteen species can interfere with 
%% reproductive signals or the transfer of gametes. 

%% When species interactions negatively effect 
%% their reproductive success, selection to 
%% reduce interference can drive displacement of their reproductive 
%% characters. 

%% 1. Interference can drive RCD
%% 2. RCD it typically studied in experimental systems
%%    with a single pair of spp. occurring in sympatry v allopatry, 
%%    yet we may expect CD to be most prevalent in highly diverse
%%    clades under going rapid radiation, where many close relatives
%%    co-occur and compete intensely. 
%% 3. Here we show that the variable composition of  louseworts (\emph{Pedicularis})
%%    fill redundant reproductive niches across a mosaic of
%%    communities, but that 

%% but that the composition of communities
%%    imposing parallel selection 
%%    pressures on the floral phenotype of a wide-spread and variable 
%%    plant species from the Hengduan mountains of China. 

%% The divergence of reproductive characters 
%% Reproductive character displacement can drive the divergence 

%% opportunity/disturbance/..: 
%% rainey & travisano 1997

%% character displacement, morphological divergence...


%% Adaptive radiation describes clades with exceptional species richness
%% and morphological disparity

%% In the study of adaptive radiation much focus has been placed
%% on the late stages of diversification, where a saturation of 
%% potential niche space[], or ecological opportunity[] limits
%% the diversification of clades, while fewer studies have examined the 
%% dynamics that accelerate and maintain elevated rates of diversification
%% in the early and mid stages. 

%% A general problem in phylogeny reconstruction is the difficulty of...
%% Part of the difficulty is
%% More fundamentally, however,...
%% In this context,

High-throughput sequencing technologies now offer much greater
potential for efficiently sampling entire genomes of any taxon for
phylogenetically informative variation. In particular,
reduced-representation methods {\citep[reviewed in][]{davey_radseq:_2010}, }
such as restriction-site associated DNA sequencing
\citep[RADseq;][]{miller_rapid_2007,baird_rapid_2008,rowe_rad_2011},
or genotyping by sequencing \citep[GBS;][]{elshire_robust_2011}, allow
the regions adjacent to restriction sites to be surveyed with deep
coverage at a high ratio of samples to sequencing effort.  These
methods are particularly appealing for systematics research because
they are easily applied to non-model organisms for which no reference
genome sequence is available. In contrast to data sets for gene tree
analysis in which sequence lengths are relatively long but the number
of sampled genes is relatively small, RADseq generates data sets of
relatively short sequences from a large number of loci. The potential
for informative gene-tree variation within a single RAD locus is thus
relatively small, but to the extent that restriction sites are
conserved across samples, the collective potential of RADseq for
detecting many single-nucleotide polymorphisms (SNPs) across the
genome is large. For this reason, it has been applied to date almost
exclusively to population-level studies \citep{baird_rapid_2008,
emerson_resolving_2010, hohenlohe_nextgeneration_2011,
baxter_linkage_2011, pfender_mapping_2011}, although attention is
now turning to its utility for phylogenetics
\citep[e.g.,][]{rubin_inferring_2012}.

In this paper, we investigate the utility of RADseq data for resolving
recalcitrant phylogenetic relationships in the angiosperm genus
\emph{Pedicularis} (Orobanchaceae, the broomrape family), which
consists of approximately 700 species of hemiparasitic herbs 
having a center of diversity in the Hengduan Mountains of south-central China
\citep{yang_flora_1998}. \emph{Pedicularis} is known for exhibiting
spectacular interspecific variation in floral traits that is thought
to reflect adaptations to reduce heterospecific pollen flow by
generalist bumble bees
\citep{macior_pollination_1983,grant_modes_1994,eaton_floral_2012}.



We focus our attention on \emph{Pedicularis} sect.\
\emph{Cyathophora}, a clade of five described species
(\emph{P.~cyathophylla}, \emph{P.~cyathophylloides}, \emph{P.~rex},
\emph{P.~superba}, and \emph{P.~thamnophila}) that are all endemic to
the Hengduan region. This clade, which is easily recognized by the
distinctive fusion of leaves around the stem, is particularly
interesting in the context of floral homoplasy, as its species
collectively exhibit almost the full range of floral variation found
across the genus as a whole (Fig.~\ref{fig:1}). Within it,
\emph{P.~rex} exhibits the most intraspecific variation, with several
described subspecies and varieties that vary not only in floral shape
and size but also color. Resolving the phylogeny of \emph{Pedicularis}
sect.\ \emph{Cyathophora} thus has the potential to yield insights
into the diversification of flowers and lineages at a relatively fine
scale that may illuminate general evolutionary patterns across the
genus as a whole.

Previous phylogenetic analyses of a nuclear marker (ITS, the internal
transcribed spacer region of nrDNA) and a chloroplast marker
(\emph{matK}) strongly supported the monophyly of \emph{Pedicularis}
sect.\ \emph{Cyathophora}, but provided little phylogenetic resolution
within it, and showed intergenomic conflict in the position of
\emph{P.~cyathophylla} \citep{ree_phylogeny_2005}. These results raise
the question of whether phylogenetic relationships have been obscured
by gene flow between species. Little is known about hybridization in
\emph{Pedicularis}: putative F1 hybrids have rarely been found in
nature, no hybrid species have been described, and the only
experimental interspecific cross reported in the literature, between
\emph{P.~longiflora} and \emph{P.~rhinanthoides}, showed
heterospecific pollen tube growth but no seed set
\citep{yang_reproductive_2007}. However, it is also the case that no
crossing experiments have been reported for closely related species
such as those in \emph{P.}~sect.\ \emph{Cyathophora}. Moreover, other
factors suggest at least the potential for interspecific gene flow. In
the Hengduan region, the many species of \emph{Pedicularis} frequently
co-occur in sympatry, flower synchronously during the short
temperate-alpine summer, and share bumble bee pollinators. The
Hengduan species are also uniformly diploid; chromosome counts show
sporophytic values of 2N = 16, and no species are reported as
polyploids or aneuploids \citep{goldblatt_index_2010}, suggesting a
lack of karyotypic barriers to interbreeding.

Here, we investigate the utility of RADseq data for inferring
phylogeny and historical introgression in \emph{Pedicularis} sect.\
\emph{Cyathophora}. As phylogenetic workflows for processing RADseq
data are not yet standard, we include a description of our software
pipeline for clustering, filtering, and sorting RAD sequences into
phylogenetically informative alignments of putative loci. We then
apply three different methods for reconstructing the phylogeny of
\emph{P.}\ sect.\ \emph{Cyathophora} using RADseq data. To test for
introgressive gene flow between lineages, we apply the D-statistic to
various four-taxon subtrees, and present a novel extension of the
D-statistic which allows investigation of the temporal sequence of
multiple introgression events. We show how this new method can be used
to infer phylogenetic relationships that are otherwise obscured by
gene flow, demonstrating this with both a simulated data set as well
as our empirical data set. We discuss the results in the context of
how floral variation and geographic ranges may have influenced
propensities for hybridization and speciation during the
diversification of \emph{P.}\ sect.\ \emph{Cyathophora}.

\section{Materials and Methods}

\subsection{Taxon sampling}
Genomic DNA was extracted from silica-dried leaf tissues of voucher
specimens collected between 2007 and 2009 in Yunnan, Sichuan and
Xizang (Tibet), China (Fig.~\ref{fig:1}). Thirteen samples represent
the five species in \emph{Pedicularis}\ sect.\ \emph{Cyathophora} as
well as the closest known outgroup species, \emph{P.~przewalskii}.
The ingroup samples include one individual of \emph{P.~cyathophylla},
one of \emph{P.~superba}, two of \emph{P.~cyathophylloides}, two of
\emph{P.~thamnophila} representing the subspecies
\emph{P.~thamnophila} subsp.\ \emph{cupuliformis} 
and \emph{P.~thamnophila} subsp.\ \emph{thamnophila}, respectively,
and five of the geographically widespread species \emph{P.~rex}. The
latter includes two individuals of \emph{P.~rex} subsp.\
\emph{lipskyana}, two of \emph{P.~rex} subsp.\ \emph{rex}, and one of
\emph{P.~rex} subsp.\ \emph{rex} var.\ \emph{rockii}. For convenience,
we refer to this last individual informally in this paper as
\emph{P.~rex} subsp.\ \emph{rockii}. 
Voucher information is available in online~Appendix~1 (doi:10.5061/dryad.bn281). %Table~S1. 

\subsection{RADseq data acquisition and analysis}
Library preparation and sequencing of RAD markers from genomic DNAs
was performed by Floragenex Inc.\ (Eugene, Oregon) using the
restriction enzyme \emph{Pst1} and sample-specific barcodes. The 13
samples studied here were pooled with 11 others and run multiplexed on
a single lane of an Illumina GAIIx sequencer for 75 cycles to 
generate single-end reads.

To process the raw RADseq data (Illumina FASTQ output files) for
phylogenetic analysis, we developed a custom software pipeline for
distinguishing sequencing errors from nucleotide polymorphisms within
samples, identifying putative orthology relationships across samples,
and assembling formatted data files. This pipeline, called
\emph{pyRAD} (http://pyrad.googlecode.com), is somewhat different from
other RADseq software packages \citep[e.g.,
\emph{Stacks};][]{catchen_stacks:_2011} that emphasize analysis of
population-level variation. Because the objective of \emph{pyRAD} is
to capture variation across species and potentially higher-level
clades, we employ a global clustering/alignment method, in contrast to
the ``off-by-N'' approach of \emph{Stacks}. This allows our method to
cluster sequences with higher levels of divergence, including indel
variation. D-statistic tests are also performed in \emph{pyRAD}.
The following sections describe the \emph{pyRAD} pipeline in more
detail.

\subsubsection{Preparing sequence files for analysis}
Given one or more Illumina sequence files in FASTQ format,
\emph{pyRAD} can de-multiplex the data and create separate files for
each sample. Sequences are identified allowing for one base mismatch
in their sample-specific barcode. The restriction site and barcode are
trimmed from each sequence, and bases with a FASTQ quality score below
a given value (here, 20) are replaced with N. Sequences having more
than a given percentage of Ns (here, 5\%) are discarded.

\subsubsection{Clustering RAD sequences within samples of genomic
  DNA}
For each sample, sequences are clustered by similarity (here, 90\%)
using the \emph{uclust} function in USEARCH \citep{edgar_search_2010}
with heuristics turned off, yielding clusters representing putative
loci.  Clusters of fewer sequences than a set minimum depth of
coverage (here, 6) are excluded in order to ensure accurate base
calls.  The remaining clusters can then either be exported to external
genotyping software or processed within \emph{pyRAD} to generate
consensus sequences.  In \emph{pyRAD}, the error rate ($\epsilon$) and
heterozygosity ($\pi$) are jointly estimated from the observed base
counts across all sites in all clusters, by applying the maximum
likelihood equation of \cite{lynch_estimation_2008}. The mean
$\epsilon$ is then used to assign consensus diploid genotypes for each
site in each cluster by calculating the binomial probability the site
is homozygous (aa or bb) versus heterozygous (ab) given the relative
frequencies of observed bases at the site and $\epsilon$
\citep{li_mapping_2008}. If a base cannot be assigned with $\ge$ 95\%
probability it is replaced by $N$ in the consensus
sequence. Heterozygotic variation is recorded using appropriate
ambiguity codes. The end result of this step is a set of consensus
sequences of putative RAD loci for each barcoded DNA sample.

\subsubsection{Clustering and filtering RAD loci across samples}
Consensus sequences from all samples are clustered by sequence
similarity, with their input order randomized, using the same
similarity threshold as in the previous step of within-sample
clustering. The resulting clusters representing putative RAD loci
shared across samples are then aligned with Muscle
\citep{edgar_muscle:_2004}. Any locus appearing heterozygous at the
same site across a set number of samples (here, 3) is discarded as
likely representing a clustering of paralogs, under the assumption
that paralogs are more likely than ancestral polymorphisms to be
shared across multiple species or samples
\citep{hohenlohe_nextgeneration_2011}.

The remaining clusters are treated as RAD loci, i.e., multiple
alignments of putatively orthologous sequences, and are assembled into
phylogenetic data matrices. For any given RAD locus, sequences of one
or more samples may be missing if substitutions in the restriction
site have disrupted recognition, or if the locus did not receive
sufficient coverage for confident basecalling. To explore the effect
of missing data, we compiled two supermatrices that differed in their
amounts of missing data: a ``minimum-taxa'' data set containing all
loci for which more than four samples were present, and a
``full-taxa'' data set containing only loci for which all 11 ingroup
samples were present.

\subsection{Simulation of RADseq data sets}
For the purpose of validating our analyses of phylogeny and
introgression in \emph{Pedicularis} sect.\ \emph{Cyathophora}, we
simulated RADseq-like data under a coalescent model with varying
degrees of interspecific gene flow. These were implemented in Python
using the EggLib library \citep{mita_egglib:_2012}. Sequences were
evolved on a phylogenetic tree of seven species (Fig.~\ref{fig:2}) to
create 30,000 loci, each 200 bp in length, using parameter values
intended to reflect the clade of herbaceous plants under study, 
including the \emph{Arabidopsis} mutation rate of 7\e{-9} 
\citep{ossowski_rate_2010} and effective population size 
N=100,000.
These values were held uniform across taxa ($\Theta$ = 4N$\mu$ =
0.0028). In one set of simulations, unidirectional gene flow occurred
from one taxon into another (C$_i$ and B$_i$, respectively;
Fig.~\ref{fig:2}) over a period of 10,000 generations, beginning
50,000 generations before the present. We created three data sets
varying in the strength of introgression (probability of migration
between taxa), with rates equal to ten migrants per generation, one
migrant per generation, and one migrant per ten generations. We refer
to these as the strong, medium and weak gene flow data sets.  In
addition, a ``multiple-reticulation'' data set was created by
simulating a scenario in which two sister taxa (C$_i$ and C) both
introgress into B$_i$ independently over the same 10,000 generation
period, at a rate equal to that of the weak gene flow
simulations. 


\subsection{Phylogenetic Inference}
To infer phylogeny from the empirical and simulated RADseq data sets
described above, we first applied a supermatrix approach in which all
RAD loci were concatenated into a single alignment, with missing data
(Ns) entered as needed for loci with incomplete taxon sampling
\citep{dequeiroz_supermatrix_2007}. Maximum likelihood trees were
inferred for the minimum-taxa and full-taxa data sets using RAxML
7.2.8 \citep{stamatakis_raxml-vi-hpc:_2006}, with bootstrap support
estimated from 200 replicate searches with random starting trees using
the GTR+$\Gamma$ nucleotide substitution model.

We also applied Bayesian concordance analysis
\citep{baum_concordance_2007} using BUCKy 1.4.0
\citep{larget_bucky:_2010}. Unlike multispecies coalescent methods,
concordance analysis is applicable to cases where gene tree
incongruence is caused by a combination of incomplete lineage sorting
and reticulate evolution (hybridization and introgression)
\citep{baum_concordance_2007,ane_bayesian_2007}. It provides estimates
of concordance factors (CFs) that measure the proportion of genes for
which a clade is true. These can be summarized by a primary
concordance tree that is composed of clades having concordance factors
greater than any contradictory clade. BUCKy also infers a population
tree that is expected to converge on the true tree when all
discordance is caused by incomplete lineage sorting. Comparing a
primary concordance tree and a population tree can thus potentially
provide insight into the influence of incomplete lineage sorting
versus introgression.

BUCKy takes as input a posterior sample of gene trees estimated for
each individual locus. For this purpose, we selected loci from the
full-taxa data set that contained at least two phylogenetically
informative SNPs, excluding sequences of the two outgroup taxa
(\emph{Pedicularis przewalskii}) and redundant individuals
representing \emph{P.\ cyathophylloides} and \emph{P.~rex} subsp.\
\emph{lipskyana}.
For each locus, we executed two independent runs of MrBayes 3.2.1
\citep{ronquist_mrbayes_2012} using the GTR+$\Gamma$ substitution
model, each run with four chains for 1,010,000 generations. These
sampled a total of 2,200 trees from the posterior distribution per
locus, of which the first 200 were discarded as burn-in. We ran BUCKy
with four chains for 500,000 generations at two different values of
$\alpha$, the prior on the number of unique gene tree topologies: 0.1
and 100. 

\subsection{Tests for introgression}

\subsubsection{Four-taxon D-statistic test}
We used the D-statistic \citep{green_draft_2010,durand_testing_2011}
to test whether introgression had occurred between two lineages in a
given phylogeny. From a pectinate topology (((P1,P2),P3),O), the
genome-wide frequencies at which two incongruent allele patterns
appear across the tips (ABBA and BABA) can be used to infer
hybridization (Fig.~\ref{fig:3}a). These patterns, in which the taxon
P3 exhibits a derived allele relative to the outgroup O that is shared
only by P1 or P2 (but not both), represent gene histories that are
incongruent with the phylogeny.  If the incongruence is caused by
stochastic lineage sorting, the frequencies of ABBA and BABA are
expected to be equal.  Alternatively, if the cause of incongruence is
introgression between P3 and either P1 or P2, the two patterns are not
expected to occur with equal frequency. The D-statistic quantifies
this asymmetry:

\bigskip
\begin{equation}
D(P1,P2,P3,O) = \frac{\sum_{i=1}^{n}C_{ABBA}(i)-C_{BABA}(i)}{\sum_{i=1}^{n}C_{ABBA}(i)+C_{BABA}(i)}. \nonumber
\end{equation}
\bigskip

\noindent Here, $C_{ABBA}(i)$ is the number of SNPs showing the ABBA
pattern in a given locus $i$, and the counts are summed over all loci.
Conservatively, we excluded sites from this test which appeared
heterozygous for any individual. To the extent that RAD sequences are
a random sample of the genome, the D-statistic represents a
genome-wide measure of introgression.

We used the D-statistic to test whether hybridization occurred within
the rex-thamnophila clade, within the superba clade, and between these
clades. In this context we refer to a ``test'' as the measurement of
the D-statistic from a distinct four-taxon subtree extracted from a
complete tree estimated in the previous step, fitting a pectinate
topology (((P1,P2),P3),O) where each tip represents a species or
subspecies. Each test in this sense can potentially sample different
individuals representing the same terminal taxon, so we define a
``replicate'' as measurement of the D-statistic from one out of all
possible combinations of individuals that could be sampled for a given
subtree.

For each replicate, we ran 1000 bootstrap iterations to measure the
standard deviation of the D-statistic, in which loci were resampled
with replacement to the same number as in the original data set. We
report the results as the range of Z-scores across all replicates in a
test, where Z is the number of standard deviations from zero (the
expected value) for D. Significance was assessed for each replicate by
converting the Z-score into a two-tailed p-value, and using $\alpha$ =
0.01 as a conservative cutoff for significance after correcting for
multiple comparisons using Holm-Bonferroni correction.

For tests showing significant introgression, we employed the equation
of \cite{durand_testing_2011} to estimate the proportion of genomic
introgression between taxa. This divides the numerator of the
D-statistic test by the numerator of an alternative test measuring the
maximum expected amount of introgression; in other words,
introgression from P3 into a close relative from the same clade
[e.g., (((P1,P3$_1$),P3$_2$),O)].

\subsubsection{Partitioned D-statistic test}

The D-statistic as described above does not take full advantage of the
information available from incongruent allele patterns in multiple
taxa. Importantly, it detects only whether alleles from one lineage
occur excessively in another lineage, but does not distinguish whether
this stems from direct gene flow from the lineage in question, or gene
flow from a close relative.
This distinction becomes increasingly important when the D-statistic
is applied at deeper or broader phylogenetic scales with redundant
sampling of taxa.
To demonstrate this problem, consider a case in which the P3 lineage
comprises sister taxa P3$_1$ and P3$_2$, and that only P3$_1$ is
sampled (Fig.~\ref{fig:3}b). If the unsampled taxon P3$_2$ hybridized
with P2, but P3$_1$ did
not, the D-statistic may falsely indicate that introgression occurred
between P3$_1$ and P2. This is a consequence of the fact that alleles
which introgressed from P3$_2$ into P2 may also be shared between
P3$_2$ and P3$_1$ due to their common ancestry exclusive of P2. In
order to distinguish these possibilities, we developed a novel
extension of the D-statistic, which we call the partitioned
D-statistic test.

The partitioned D-statistic is applied as follows when two lineages
from within the P3 clade have been sampled. These lineages, P3$_1$ and
P3$_2$, are assumed not to have exchanged genes with each other. As
before, we measure asymmetry in the counts of incongruent sites, with
the key difference being that we now ask whether a derived allele is
present in only P3$_1$ and not P3$_2$, in P3$_2$ and not P3$_1$, or
whether it is shared by both (Fig.~\ref{fig:3}b). This amounts to
measuring three D-statistics, one for each scenario, which we denote
D$_1$, D$_2$ and D$_{12}$, respectively. These are calculated in the
same manner as the original D-statistic, but are based on 5-taxon
allele patterns. For example, for the species tree topology
(((P1,P2),(P3$_1$,P3$_2$)),O), D$_{1}$ takes as input the counts of
ABBAA and BABAA, and measures the signal of introgression involving 
P3$_1$, while D$_{12}$ takes as input the counts of ABBBA and BABBA,
and measures the signal of introgression involving the branch subtending
the most recent common ancestor of P3$_1$ and P3$_2$.

To verify that the partitioned D-statistic can indeed distinguish
single or multiple introgression events from the signal of shared
ancestry, we applied it to two simulated data sets: the weak gene flow
data set and the multiple-reticulation data set, which differ in
whether one (C$_i$) or two (C$_i$ \& C) taxa are the source of
introgression into another taxon (B$_i$).

\subsubsection{Directionality of gene flow}
While the four-taxon D-statistic cannot distinguish the directionality
of gene flow, i.e., whether it occurred from P2 into P3, from P3 into
P2, or in both directions, the partitioned D-statistic can infer
directionality through its measurement of introgression of shared
ancestral alleles, D$_{12}$.  For example, if gene flow occurred from
P3$_1$ into P2, then derived P3 alleles which arose in the ancestor of
P3$_1$ and P3$_2$, and are thus shared by both taxa, will also appear
in P2.  By contrast, if gene flow occurred only in the opposite
direction, from P2 into P3$_1$, P2 will not contain alleles that 
are shared by the two P3 taxa, and thus the partitioned test would
find a non-significant D$_{12}$.
In this way, D$_{12}$ acts as an indicator, showing whether 
introgression occurred from the P3 lineage into P2, versus whether
the signal is caused by gene flow in the opposite direction. Contrast
this with the four-taxon test, where a significant D for tests (P1,P2,P3$_1$,O),
(P1,P2,P3$_2$,O) or (P3$_1$,P3$_2$,P2,O) would all indicate introgression, but
fail to distinguish that only P3$_1$ and not P3$_2$ introgressed into P2 
(which D$_1$ vs. D$_2$ would indicate), or that introgression occurred in only one
direction (which D$_{12}$ indicates).}
This is demonstrated in the Results section using the simulated data sets.

\subsection{Detecting errors in total-evidence tree reconstruction
 due to introgression}

\subsubsection{Partitioned D-statistic}
By separating the signal of introgression into components attributable
to shared versus independent ancestry, the partitioned D-statistic may
also be used to test whether P3$_1$ and P3$_2$ are indeed monophyletic
relative to P1, P2, and O. 
The reasoning is that if P3$_1$ and P3$_2$ are paraphyletic with respect
to P1 and P2, as in the case of ((((P1,P2),P3$_1$),P3$_2$),0), then 
D$_{12}$ will not deviate significantly from zero, because the P3 taxa 
do not share any common history independent of P1 and P2.
We applied the partitioned D-statistic test in this way to investigate
the relationship of the two subspecies of \emph{P.~thamnophila} with
respect to the subspecies of \emph{P.~rex}.

\subsubsection{Censored comparisons of alternative topologies}
We used the Shimodaira-Hasegawa test \citep[SH
test;][]{shimodaira_multiple_1999} as implemented in RAxML to test
whether total-evidence reconstructions of phylogeny might be
positively misled by introgression, e.g., if P3$_1$ and P3$_2$ are in
fact sister taxa but appear paraphyletic because one (e.g., P3$_1$)
has introgressed with another lineage (e.g., P2). The procedure first
applies the SH test to topologies with full taxon sampling,
comparing a topology in which the taxa of interest are paraphyletic
with a topology that differs only in having them monophyletic.  The SH
test is then repeated after removing from the data set one of the taxa
suspected to have undergone introgression (e.g., P2). SH tests are
thus always made between trees containing the same set of taxa. The
rationale is that when one of a pair of reticulating taxa is removed
from the analysis, introgressed sites which would otherwise be treated
as synapomorphies (shared by the taxa, with the effect of pulling them
erroneously together) will instead appear as autapomorphies in the
remaining taxon, erasing the effect of introgression on the inferred
topology. In other words, the objective is to ``censor'' the effect of
introgressed DNA on phylogenetic inference.

To investigate the monophyly of \emph{P.~thamnophila}, we first
applied the SH test before and after removing all samples of
\emph{P.~rex} other than \emph{P.~rex} subsp. \emph{rockii}. Then, we
applied the SH test to compare the relationships among the three
subspecies of \emph{P.~rex} before and after removing each subspecies
of \emph{P.~thamnophila}. 

\section{Results}

\subsection{RAD data and processing}
Illumina sequencing returned an average of 1.35\e{6} reads per sample,
which after filtering and clustering was reduced to an average of
48,000 clusters \citep[or ``stacks'', following the terminology
of][]{emerson_resolving_2010} with coverage greater than our set
minimum of six, yielding a mean coverage depth of 19.80. Consensus
sequences were called for each cluster, yielding approximately 45,000
loci per sample (Table~\ref{tab:1}).  The ML estimate of the
sequencing error rate is lower than heterozygosity
($\epsilon$=2.3\e{-3} and $H$=6.7\e{-2}, respectively), and both
values are within the range where simulation showed they could be
accurately estimated (results not shown). The last five bases were
trimmed from all loci, as the error rate was found to increase
precipitously in this region, giving a final average read length of 65
bp.


Clustering of consensus sequences across all 13 samples revealed
268,901 unique clusters. %(Supplemental Figure~X).
The minimum-taxa data set (loci with at least four samples) contained
42,235 loci and a total of 61,829 phylogenetically informative
sites. The full-taxa data set (all loci have complete sampling of the
11 ingroup individuals) contained 4,837 loci and 8,227
phylogenetically informative sites. The proportion of missing data in
each case was 37\% and 7.6\%, respectively. The occurrence of each
sample in the final data sets was relatively uniform, with the
outgroup samples being recovered least often (Table~\ref{tab:1}). The
BUCKy data set included 945 loci containing at least two
phylogenetically informative sites among the nine selected ingroup
samples.
 
\subsection{Phylogeny reconstruction}
The ML, primary concordance, and population trees for
\emph{Pedicularis} sect.\ \emph{Cyathophora} were all congruent with
the ITS topology of \cite{ree_phylogeny_2005} in recovering the
rex-thamnophila and superba clades, with \emph{Pedicularis
  cyathophylla} nested within the latter. Resolution within clades was
less certain, as described below. From simulated data, the correct
topology was recovered for the low and medium gene flow data sets, but
not in the strong gene flow data set.

ML analysis of the minimum-taxa (sparse) and full-taxa (dense)
supermatrices yielded topologies with comparable branch lengths, but
which differed in their resolution of the three subspecies of
\emph{P.~rex} (Fig.~\ref{fig:4}a-b). In both trees, the subspecies of
\emph{P.~thamnophila} were paraphyletic.  The minimum-taxa data set
gave high bootstrap support for all clades, whereas the full-taxa tree
had lower support for shorter branches. For all the simulated data
sets, including the strong gene flow data set, inferred ML trees had
uniform bootstrap support of 100\% for all branches.

For the Bayesian concordance analysis we report CFs as their mean and
95\% confidence intervals (CI) on the primary concordance trees, and
as the quartet CF for population trees. CFs in the primary concordance
tree that do not overlap in their CI with any conflicting clade are
considered significantly supported.  A full account of the BUCKy
analyses is provided in online~Appendix~2. %Table~S2.

Concordance factors provided an accurate measure of genomic
reticulation, as evidenced by the correlation between CFs and the
amount of interspecific gene flow in simulated data sets. For example,
the true clade (D,(C,C$_i$)) was recovered with CFs of 0.99, 0.87, and
0.12 for the low, medium, and strong gene flow data sets,
respectively. In the strong gene flow data set, an erroneous clade
composed of the two reticulating taxa (B$_i$,C$_i$) is present in the
primary concordance tree with a CF of 0.75.  Population trees matched
the primary concordance tree topologies for all three simulated data
sets. In all cases the value of $\alpha$ had little effect on the
results.

Primary concordance trees inferred from the empirical data recovered a
wide range of CFs, with some clades having high support, and others
showing evidence of reticulation. In all cases, the superba clade
(\emph{P.~cyathophylloides}, (\emph{P.~cyathophylla},
\emph{P.~superba})) was well-resolved with significant CFs
(Fig.~\ref{fig:4}c-d). However, conflicting relationships in this
clade were also evident: the clade (\emph{P.~cyathophylla},
\emph{P.~cyathophylloides}) had a CF of 0.21, and the clade
(\emph{P.~superba}, \emph{P.~cyathophylloides}) had a CF of
0.07. These results were independent of $\alpha$. The asymmetry in CF
values suggests that reticulation, as opposed to incomplete lineage
sorting alone, may have occurred. This comparison is in 
principle analogous to the D-statistic \citep{ane_reconstructing_2010,chung_comparing_2011},
utilizing gene tree heterogeneity as opposed to allele frequencies.

The value of $\alpha$ had a greater effect on results in the
rex-thamnophila clade (Fig.~\ref{fig:4}c-d), where conflict was more
evident.  When $\alpha$ was low (0.1), the only clade with significant
support grouped the two sampled populations of \emph{P.~rex}
subsp. \emph{rex}. In other clades found with low support,
\emph{P.~thamnophila} was either monophyletic or paraphyletic, with
\emph{P.~thamnophila} subsp. \emph{thamnophila} grouping with either
\emph{P.~rex} subsp. \emph{rex} or \emph{P.~rex}
subsp. \emph{lipskyana}.  When $\alpha$ was high (100), the monophyly
of \emph{P.~thamnophila} was significantly supported
(Fig.~\ref{fig:4}d), and relationships among the three subspecies of
\emph{P.~rex} remained unresolved.

In contrast to the simulation results, population trees of
\emph{Pedicularis} sect.\ \emph{Cyathophora} did not match the primary
concordance trees. Most notably, the population trees matched the ML
trees in that \emph{P.~thamnophila} was not monophyletic, with
\emph{P.~thamnophila} subsp. \emph{thamnophila} consistently being
placed as sister to \emph{P.~rex} subsp. \emph{rex}
(Fig.~\ref{fig:4}e-f), and \emph{P.~thamnophila}
subsp. \emph{cupuliformes} being placed as sister to the rest of the
rex-thamnophila clade.

\subsection{Tests for introgression}

\subsubsection{D-statistic tests: simulated data}
The four-taxon D-statistic test accurately detected introgression in
all simulations where it occurred (tests 1.1 and 1.2;
Table~\ref{tab:2}), and rejected introgression where it did not
(tests 2.1--2.10). In some cases, however, significant
introgression was falsely detected (3.1--3.10). These include tests
where P3 shares fewer derived alleles with P1 relative to P2 because
P1 received introgressed DNA from a more distant clade (tests
3.1--3.3). As an example, taxon A shares more derived alleles
with taxon B relative to B$_i$ in test 3.1 not as a consequence of
having introgressed with B, but rather because B$_i$ received 
alleles from the more distant taxon C$_i$. %\textbf{CONFUSING}} 
A false positive is also detected
when the P3 taxon is a close relative of the true source of
introgression (C$_i$), such that the signal picked up is merely the
proportion of P3's genome that is shared with C$_i$ through common
ancestry (tests 3.4-3.7). While these do not represent true false
positives in the sense that introgression did actually occur from a
lineage that P3 is a member of, it is false in the sense that
introgression is still detected whether or not the true hybridizing
taxon is sampled; thus, the test would lead to the conclusion that
both P3 taxa had experienced introgression with P2.  Finally, tests
where B$_i$ is in the P3 position (tests 3.8-3.10), such that
introgression is being tested in the opposite direction it actually
occurred, also yielded a positive result.

The partitioned D-statistic test, by contrast, accurately detected the
presence or absence of introgression in all simulated data sets,
including those scenarios which led to false positives in the
four-taxon test. When introgression occurred only from P3$_1$ and not
P3$_2$, a significant value of D$_1$ and non-significant value of
D$_2$ were correctly obtained (tests 4.1 - 4.4; Table~\ref{tab:3}). In
the multiple-reticulation data set, where introgression occurred from
both P3 lineages independently, both D$_1$ and D$_2$ were significant
(tests 4.3 \& 4.5). These results highlight the importance of sampling
multiple lineages in order to more precisely identify introgressing
taxa. For example, if C$_i$ had not been sampled, such that C was the
only representative of (C,C$_i$), then the partitioned D-statistic test
would indicate that C introgressed with B$_i$ (tests 4.1 \& 4.2). Only
by having more sampled lineages in (C,C$_i$) are we able to repeat the
test at more shallow nodes in the phylogeny (tests 4.3 \& 4.5), which
yields the results that introgression occurred only from taxon C$_i$
since its divergence from C.

The partitioned D-statistics accurately rejected introgression in
tests where it did not occur (tests 5.1-5.5), and correctly indicated
the direction of introgression when it did occur.  For example, a
significant D$_1$ was detected when B$_i$, the recipient of
introgressed DNA, was tested in the P3 position (tests 6.1-6.6). 
However, because the introgressed alleles in these cases are unique to 
the P3$_1$ lineage (B$_i$), having come from P2 (C$_i$), they are not shared 
between multiple members of the P3 clade by ancestry (e.g., between B$_i$ and A or B). 
Thus, the value of D$_{12}$ does not deviate significantly from zero.

\subsubsection{Four-taxon D-statistic tests: empirical data}
Uncertainty in the topology of the rex-thamnophila clade arising from
the BUCKy analysis yielded a large number of distinct four-taxon
subtrees on which to perform D-statistic tests. We present selected
results from these tests below, with the full list available in
online~Appendix~3. Test results are reported as ranges of Z-scores from
multiple replicates, where a single replicate constitutes a unique
sampling of redundant individuals within taxa for a given subtree. The
number of RAD loci for which data were available across all four taxa
in a test ranged from about 5,000 to 21,000, of which 1--5\%
contained at least one informative discordant site. More loci were
available in tests performed among closer relatives.

We first tested whether introgression occurred between the
rex-thamnophila and superba clades, using \emph{P.~przewalskii} as the
outgroup. These yielded no evidence of introgression (tests 8.1-8.6;
Table~\ref{tab:4}). Next, we tested for introgression within each
clade, using members of the other clade as the outgroup. No
significant results were detected between any members of the superba
clade (tests 9.1-9.3).  The two samples of
\emph{P.~cyathophylloides} have insufficient differences to detect
introgression into one versus the other (tests 9.2 \& 9.3).
\emph{Pedicularis cyathophylloides} shares more derived alleles with
\emph{P.~cyathophylla} than with \emph{P.~superba} (test 9.1),
consistent with the concordance factor results, which showed a
greater CF for (\emph{P.~cyathophylloides}+\emph{P.~cyathophylla})
relative to (\emph{P.~cyathophylloides}+\emph{P.~superba}).
However, the difference is non-significant, suggesting incomplete
lineage sorting alone may be sufficient to explain this result. 

By contrast, within the rex-thamnophila clade nearly all individuals
showed significant evidence of introgression. Given a test topology in
which the two subspecies of \emph{P.~thamnophila} are paraphyletic in
positions P3 and P2, \emph{P.~thamnophila} subsp.\ \emph{cupuliformes}
showed significant introgression with \emph{P.~thamnophila} subsp.\
\emph{thamnophila} relative to any sample of \emph{P.~rex} in position
P1 (test 10.1). Testing each subspecies of \emph{P.~thamnophila}
separately in position P3, with two samples from \emph{P.~rex} in
positions P1 and P2, we found that \emph{P.~thamnophila} subsp.\
\emph{cupuliformes} may have introgressed with \emph{P.~rex} subsp.\
\emph{rex} relative to \emph{P.~rex} subsp.\ \emph{rockii} (test
10.3), while \emph{P.~thamnophila} subsp.\ \emph{thamnophila} appears
to have introgressed with both \emph{P.~rex} subsp.\ \emph{rex} and
\emph{P.~rex} subsp.\ \emph{lipskyana} relative to \emph{P.~rex}
subsp.\ \emph{rockii} (tests 10.6 \& 10.7). Neither subspecies of
\emph{P.~thamnophila} showed greater introgression with \emph{P.~rex}
subsp.\ \emph{rex} relative to \emph{P.~rex} subsp.\ \emph{lipskyana}
(tests 10.2 \& 10.5).

Given a topology in which the two subspecies of \emph{P.~thamnophila}
are monophyletic, occupying positions P1 and P2, tests placing each
subspecies of \emph{P.~rex} in position P3 showed very strong
introgression with \emph{P.~thamnophila} subsp.\ \emph{thamnophila} 
relative to \emph{P.~thamnophila} subsp.\ \emph{cupuliformes}
(tests 10.8--10.10).

\subsubsection{Partitioned D-statistic tests: empirical data}
The partitioned D-statistic test could only be performed using RAD
loci containing sites with incongruent allele patterns across the five
taxa being tested. Fewer sites met this criterion than for the
four-taxon test, so statistical power was comparatively limited. In
some cases, fewer than 100 sites for each allele pattern were
available (online~Appendix~3). In contrast to the four-taxon test, which is
agnostic about the directionality of introgression, the partitioned
D-statistic allows the direction to be explictly tested.

Treating the two subspecies of \emph{P.~thamnophila} as monophyletic
in positions P3$_1$ and P3$_2$, test 11.1 (Table~\ref{tab:5}) suggests
introgression may have occurred between the \emph{P.~thamnophila}
clade and \emph{P.~rex} subsp.\ \emph{rex} relative to \emph{P.~rex}
subsp.\ \emph{rockii}, as evidenced by a significant
D$_{12}$.  However, D$_1$ was significant in only one of
eight replicates, and D$_2$ was consistently non-significant,
suggesting that introgression did not occur from either subspecies of
\emph{P.~thamnophila} since their divergence from each other.  This
result could be interpreted in one of two ways. First, introgression
may have occurred predominantly from \emph{P.~rex} subsp.\ \emph{rex}
into both subspecies of \emph{P.~thamnophila} (or their ancestor).
This would explain how the latter both exhibit alleles that are
derived in \emph{P.~rex} subsp.\ \emph{rex} relative to \emph{P.~rex}
subsp.\ \emph{rockii}, but \emph{P.~rex} subsp.\ \emph{rex} does not
to contain any alleles derived uniquely in either subspecies of
\emph{P.~thamnophila}.  Alternatively, introgression may have occurred
into \emph{P.~rex} subsp.\ \emph{rex} from an 
unsampled lineage which diverged from the ancestor of the 
two sampled \emph{P.~thamnophila} subspecies, or from their direct
ancestor if it occurred before their divergence. Either scenario would
yield a significant D$_{12}$ but non-significant D$_1$ and D$_2$. 
Additional sampling of \emph{P.~thamnophila} will be necessary to
further clarify this result.  Tests 11.2 and 11.3 similarly show
non-significant or weak signals of introgression from
\emph{P.~thamnophila} into the other subspecies of \emph{P.~rex}.

For all tests placing subspecies of \emph{P.~rex} in positions P3$_1$
and P3$_2$, significant introgression was detected into
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} when tested relative
to \emph{P.~thamnophila} subsp.\ \emph{cupuliformes} (tests
12.1-12.3).  Moreover, in addition to a very significant D$_{12}$, all
tests have a significant D$_1$ and D$_2$, suggesting that all
three subspecies of \emph{P.~rex} have introgressed into
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} independently since
their divergences from one another.

Focusing closer to the tips of the phylogeny, and
considering %replicates of
the test placing the two samples of \emph{P.~rex} subsp.\ \emph{rex}
in the P3 clade (and therefore testing for introgression occurring
after their divergence from each other), we find significant
introgression from only one sample of \emph{P.~rex} subsp.\ \emph{rex}
into \emph{P.~thamnophila} subsp.\ \emph{thamnophila}, suggesting this
event occurred very recently (test 12.4).  Similarly, introgression
also occurred from one of the two samples of \emph{P.~rex} subsp.\
\emph{lipskyana} since their even more recent divergence (test
12.5). In both cases, the introgressing population is the one located
geographically closer to the sampled population of
\emph{P.~thamnophila} subsp.\ \emph{thamnophila}.  (``Rr$_2$'' and
``Rl$_2$''; Fig.~\ref{fig:1}).

\subsubsection{The proportion of genomic introgression}

In simulated data, the mean proportion of genomic introgression from
C$_i$ into B$_i$ was accurately estimated to be 1\%, 10\%, and 90\%
for the weak, medium and strong gene flow data sets, respectively,
corresponding with the simulation parameters.  However, as with the
four-taxon D-statistic, we find this test is similarly biased by
shared ancestry. For example, recall that in these simulations, taxon
C did not itself introgress with B$_i$, but was a close relative of
the taxon that did, C$_i$; yet the mean proportion of genomic
introgression from C into B$_i$ was estimated to be 0.09, 8.7, and
87\%.  Moreover, in the multiple-reticulation data set, which
includes two taxa (C and C$_i$) independently introgressing into B$_i$
at the same rate as in the weak gene flow data set (0.01), we measured
the mean proportions of genomic introgression from C and C$_i$ each to
be 0.02, twice the expected value, showing an additive effect of
introgression from multiple closely related lineages.

Using this same method, we calculated the proportion of genomic
introgression among members of the rex-thamnophila clade (online 
Appendix 3). Genomic introgression from \emph{P.~rex} subsp.\ \emph{rex} into
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} is estimated to be
26.6\%, and that from \emph{P.~rex} subsp.\ \emph{lipskyana} into
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} as 8.7\%.
Although this is meant to provide a minimum estimate, the additive
nature of this measurement when hybridization occurs from multiple
taxa, coupled with our results showing that all three subspecies of
\emph{P.~rex} introgressed into \emph{P.~thamnophila}
subsp.\ \emph{thamnophila} independently, leads us to suspect these
estimates are inflated.

\subsection{Detecting errors in tree reconstruction due to
  introgression}

\subsubsection{Partitioned D-statistic}
In the simulated data sets the signal of introgression measured by
D$_{12}$ proved an accurate indicator of monophyly versus paraphyly of
taxa P3$_{1}$ and P3$_{2}$, by detecting a non-significant D$_{12}$ in
all cases for which paraphyletic species were grouped as a P3 clade
(tests 7.1-7.4). In the empirical data, a significant D$_{12}$ was
detected in one out of eight tests involving the two subspecies of
\emph{P.~thamnophila} in the P3 position. However, as we noted above,
this signal could have been caused by introgression in the opposite
direction.  Alternative topologies where \emph{P.~thamnophila} subsp.\
\emph{thamnophila} is nested within \emph{P.~rex} received no support,
all having non-significant D$_{12}$ values (tests 13.1--13.4).

\subsubsection{Censored comparisons of alternative topologies}
With complete taxon sampling, the SH test 
shows significant support for the
unconstrained ML topology in which \emph{P.~thamnophila} is
paraphyletic, compared to the constrained topology in which it is
monophyletic ($\Delta ln$L = 198.90, p < 0.05). 
However, after removing \emph{P.~rex} subsp.\ \emph{rex} and \emph{P.~rex} subsp.\
\emph{lipskyana}, leaving only \emph{P.~rex} subsp.\ \emph{rockii}
(Fig. 5a-b), monophyly of \emph{P.~thamnophila} is favored, 
though the unconstrained topology cannot be significantly rejected 
($\Delta ln$L = 85.82, p > 0.05).
Applying the same approach reveals the effect of selective taxon
removal on relationships within \emph{P.~rex}. With complete sampling,
the ML topology ((\emph{P.~rex} subsp.\ \emph{rockii}, \emph{P.~rex}
subsp.\ \emph{lipskyana}), \emph{P.~rex} subsp.\ \emph{rex}) was
favored over a constrained topology ((\emph{P.~rex} subsp.\
\emph{rex}, \emph{P.~rex} subsp.\ \emph{lipskyana}), \emph{P.~rex}
subsp.\ \emph{rockii}) ($\Delta ln$L = 420.22, p > 0.05). After
removing each subspecies of \emph{P.~thamnophila} from the data set
(Fig.~\ref{fig:5}c-e), the constrained topology in which \emph{P.~rex}
subsp.\ \emph{rockii} is sister to the other two subspecies is a
significantly better fit to the data than the two alternatives
($\Delta ln$L = 240.60, 252.89; p < 0.01).

\section{Discussion}

\subsection{Phylogenetic inference using RADseq data}
In this study we use RADseq data to resolve shallow phylogenetic
relationships and test for introgression in a systematically
problematic clade of flowering plants, \emph{Pedicularis} sect.\
\emph{Cyathophora}. In comparison to previous studies, which used
sequence data sets of at most a few markers, each on the order of 1
kbp in length, the RADseq data set is notable for its size, having
more than 40,000 loci and 60,000 potentially informative sites in the
largest supermatrix. 
As workflows for processing RADseq data for
phylogenetic analysis are not yet widely available, we developed new
software (pyRAD) for this purpose.

As sequencing technology improves and becomes more cost-effective,
reduced-representation genomic data sets are rapidly becoming
attainable for non-model organisms; 
indeed, since we began this study
up to 4X the output can now be produced for a similar cost, 
allowing greater multiplexing of individuals
and deeper coverage. 
In this context, the question of
how such data can be used to reconstruct the tree of life comes
increasingly to the fore. A common approach applies the principle of
total evidence, namely that phylogeny should be reconstructed from as
much data as possible, with the aim of identifying the dominant signal
\citep[e.g.,][]{kluge_concern_1989}. A contrasting approach asserts
that phylogenetic signal should be investigated on a gene-by-gene
basis, with the causes of gene tree incongruence in mind
\citep[e.g.,][]{rannala_phylogenetic_2008}. RADseq data are easily
concatenated for use in total-evidence tree inference, as demonstrated
in our ML supermatrix analyses. However, the data are less suited for
gene tree approaches such as concordance analysis, because individual
RAD loci as assembled here are limited in sequence length, contain
relatively few variable sites, and do not generally yield resolved
gene trees. As a result, a large proportion of loci are necessarily
excluded from consideration, and selectively choosing the most variable 
RAD loci poses a risk of introducing potential biases; 
for example, if these regions are more variable as a result
of retaining ancestral polymorphisms or by more frequently 
representing regions of introgressed DNA. 

Improvements in next-generation sequencing methods have
  the potential to greatly improve the utility of RADseq and related
  approaches to phylogenetic studies. Among these, paired-end Illumina
  sequencing may yield the most immediate and significant benefits.
  For example, it could double the length of reads produced using GBS
  methods (in which both ends of a DNA fragment contain restriction
  enzyme recognition sites), yielding 200-300 bp of sequence data per
  locus. For RADseq protocols, in which fragment size selection is
  performed by random shearing, even greater sequence lengths are
  achievable, because long contigs can be assembled from partially
  overlapping sheared-end reads. For example, \cite{etter_local_2011}
  used this method to assemble contigs up to several hundred bp in
  length.

\subsection{Detecting introgression}
Using simulations, we showed that on a four-taxon subtree, Patterson's
D-statistic test can have a high rate of type-1 error in detecting
introgression between taxa in positions P2 and P3, because it does not
discriminate between incongruent allele patterns that arise directly
from hybridization of the specific taxa sampled in the test, and
patterns that would arise if hybridization had occurred between one of
the sampled taxa and a close relative of the other.
Given a taxon in position P2 and multiple candidate taxa in position
P3, the partitioned D-statistic test can be used to more precisely
identify which of the P3 taxa contributed to introgression, 
under the assumption that the two P3 candidates have 
not exchanged genes with each other.
By distinguishing between introgressed alleles that are
unique to individual taxa in P3 and
those that are shared by common ancestry, the test can reveal the
timing of introgression relative to phylogenetic divergence events in
the P3 clade. The partitioned D-statistic test is thus a novel
extension of the method that improves its utility above the species
level.

While D-statistic tests were originally applied to test ancient  
admixture between now extinct and modern human populations \citep{green_draft_2010, 
skoglund_archaic_2011,meyer_high-coverage_2012} they have more recently been applied 
at deeper phylogenetic scales and within more diverse clades \citep{the_heliconius_genome_consortium_butterfly_2012},
where teasing apart introgression from the signal of shared ancestry is
of increased importance. 
This is especially true when the P3 lineage contains multiple distinct 
species or eco-morphs. A failure to distinguish whether introgression 
occurred independently from each P3 taxon following their divergence from each other 
could yield false positives that would ultimately inflate estimates
of the frequency of natural hybridization.

Since its original description other extensions of the D-statistic 
methodology have also been developed.
In particular, \cite{meyer_high-coverage_2012} 
described an ``enhanced D-statistic'', which involves performing the four-taxon 
D-statistic across only a subset of sites for which multiple sampled individuals 
of the P1 taxon -- in their case 30 human populations from sub-Saharan Africa
-- are all fixed for the ancestral allele. By effectively removing sites 
where the derived allele differs between P1 and P2 due to sorting of ancestral 
polymorphisms, this method increases the signal to noise ratio, 
enhancing the signal of introgression. 
In contrast to the partitioned D-statistic which aims to improve
the performance of D-statistics at deeper phylogenetic scales, the
enhanced test is most effective at shallow scales, such as among 
recently diverged populations, where ancestral polymorphisms are common. 

\cite{meyer_high-coverage_2012} similarly described new methods
for estimating the proportion of introgressed DNA
between groups, including under scenarios where multiple P3 taxa
could serve as the source of introgression. 
Their method is specifically tailored to the Neanderthal 
and Denisovan data sets, however, where other historical information
allows assumptions about the order in which gene flow is likely 
to have occurred. In other words, they assume Neanderthal gene flow
occurred first into all non-African populations, and thus Denisovan 
gene flow can be detected by measuring the excess signal of 
archaic ancestry over that expected to be present in all 
humans outside of Africa. This method may not be 
suitable to all data sets, and we propose that the partitioned 
D-statistic provides a more simple and general test to distinguish 
introgression events from among multiple P3 taxa. 

All previous studies applying the D-statistic have
utilized a reference genome, which provides linkage information and 
longer stretches of DNA from which to measure
variation in the distribution of incongruent sites. 
To measure sampling error of D-statistics, as well as their 
significance, the asymmetry in incongruent allele patterns 
is generally assessed through a block jack-knife 
approach \citep{green_draft_2010}, splitting the genome 
into a set number of blocks and removing 
them sequentially. Without access to a full genome alignment
or other linkage information, we implemented a
modification on this strategy based on the theoretical distribution
of RADseq data: to the extent RAD loci represent a random distribution
of unlinked markers from across the genome, bootstrap
re-sampling should provide an accurate measure of the
genome-wide variation in incongruence. Previous studies reported 
no significant effect of jack-knife block size on D-statistic results
\citep{meyer_high-coverage_2012}, and our implementation of the
bootstrap method to simulated RADseq data accurately
detected introgression. Because short-read
\emph{de-novo} RADseq loci, such as we use here, 
appear sufficient for detecting genome-wide patterns of 
introgression, the application of D-statistic tests could
be expanded more broadly, including within diverse groups 
or organisms that yet lack a reference genome or linkage map. 
The simulations presented here are limited in scope,
however, and further studies will be needed to evaluate how different 
methods and data types affect D-statistic results.

Analyses of D-statistics across a range of four- and five-taxon subtrees of
\emph{Pedicularis} sect.\ \emph{Cyathophora} revealed clear evidence
of recurrent introgression among taxa in the rex-thamnophila clade,
with all sampled subspecies of \emph{P.~rex} appearing to have exchanged
genes at some point with \emph{P.~thamnophila} subsp.\
\emph{thamnophila} relative to \emph{P.~thamnophila} subsp.\
\emph{cupuliformes}. The partitioned D-statistic test showed that for
both \emph{P.~rex} subsp.\ \emph{rex} and \emph{P.~rex} subsp.\
\emph{lipskyana}, only one of the two populations sampled in each case
yielded a signal of introgression. This suggests that these most
recent introgression events occurred since the divergence of the two
populations of each subspecies, and may have been localized to
particular geographic locations (discussed in more detail below).

\subsection{The effect of introgression on phylogenetic inference}

Detecting introgression using the D-statistic presents something of a
paradox, in that some knowledge of the ``true'' phylogeny --
minimally, a pectinate four-taxon subtree -- is required to formulate
a hypothesis, while the process of introgression itself acts to
obscure those relationships. This motivated us to explore alternative
means of assessing whether phylogenetic inference might be positively
misled by reticulation events. Using the SH test, we compared the ML
total-evidence phylogeny of \emph{Pedicularis} sect.\
\emph{Cyathophora} with alternative topologies before and after
removing selected taxa in order to effectively eliminate the influence
of introgressed alleles on tree inference. This revealed two cases in
which the total-evidence topology could be inaccurate: first, the
paraphyly of \emph{P.~thamnophila} (Fig.~\ref{fig:4}a,b), and second,
the position of \emph{P.~rex} subsp.\ \emph{rockii} as sister to
\emph{P.~rex} subsp.\ \emph{lipskyana}. Censored SH tests show that,
if the signal of introgression is removed, the most likely topology
has \emph{P.~thamnophila} being monophyletic, and \emph{P.~rex}
subsp.\ \emph{rockii} as sister to the other subspecies of
\emph{P.~rex} (Fig.~\ref{fig:6}a).

If this tree is correct, such that \emph{P.~thamnophila} subsp.
\emph{thamnophila} and \emph{P.~thamnophila} subsp.\
\emph{cupuliformes} do share a most recent common ancestor, a
signal of this ancestor may be preserved in introgressed alleles.
This idea stems from our simulation results, and makes sense logically
under a unidirectional gene flow model, where a significant D$_{12}$
would be detected if the two P3 samples shared a unique history
over which share derived alleles could arise.  In support of
\emph{P.~thamnophila} monophyly, tests treating the two subspecies as
monophyletic detected a significant D$_{12}$ showing introgression
into \emph{P.~rex} subsp.\ \emph{rex} (test 11.1); whereas no
alternative tests grouping \emph{P.~thamnophila} subsp.\
\emph{thamnophila} with a \emph{P.~rex} taxa recovered a significant
D$_{12}$ (tests 13.1-13.4).  Further studies are needed to investigate
the situations under which D$_{12}$ is informative about ancestral
relationships, including complex scenarios including multiple
pairs of introgressing taxa, and bi-directional gene flow.

\subsection{Temporal and spatial sequences of gene flow}
Of the many factors that may influence the likelihood of gene flow
between populations, geographic proximity is among the most important
\citep{jenkins_meta-analysis_2010}. The phylogenetic signal of
introgression is thus potentially informative about historical biogeography
to the extent that it reflects ancestral contact zones of geographic
ranges. With this in mind, we can approach the results of our
investigation of \emph{Pedicularis} sect.\ \emph{Cyathophora} with the
objective of piecing together the temporal and spatial sequences of
introgression, geographic isolation, and lineage divergence, which we
present as a hypothetical reconstruction (Fig.~\ref{fig:6}b).

\emph{P.~rex} subsp.\ \emph{rockii} is of particular interest from
this perspective. Its geographic range is currently isolated to the
south of all other lineages in the rex-thamnophila clade
(Fig.~\ref{fig:1} \& Fig.~\ref{fig:6}b). Results from censored SH
tests and partitioned D-statistic tests indicate that it was the first
to branch from the ancestral \emph{P.~rex} lineage, and that it
introgressed into \emph{P.~thamnophila} subsp.\
\emph{thamnophila}. This suggests that before \emph{P.~rex} subsp.\
\emph{rockii} became geographically isolated, the ranges of these taxa
were in contact. Given that \emph{P.~rex} subsp.\ \emph{rockii} shows
no evidence of gene flow with either \emph{P.~rex} subsp.\ \emph{rex}
or \emph{P.~rex} subsp.\ \emph{lipskyana}, it seems likely that
\emph{P.~rex} subsp.\ \emph{rockii} became geographically isolated
before the others diverged, i.e., during time 2 in
Fig.~\ref{fig:6}, and thus gene flow into \emph{P.~thamnophila}
subsp.\ \emph{thamnophila} likely occured during this time.


The next introgression event we identified with confidence,
occurring into \emph{P.~thamnophila} subsp.\ \emph{thamnophila} from 
\emph{P.~rex} subsp.\ \emph{rex} and \emph{P.~rex} subsp.\
\emph{lipskyana}, is more difficult to date precisely. From our data,
we can typically only detect the most recent introgression events, 
which in this case appear to have occurred very recently
(time 4), following the divergence of sampled populations within each
subspecies of \emph{P.~rex}. In both cases, introgression occurred
from the geographically more proximate population to the
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} sample. This does not
preclude, however, one or more earlier hybridization events,
which we cannot detect without additional samples from 
each clade to use for comparison. If gene flow indeed occurred very
recently, it is interesting to note that \emph{P.~rex} subsp.\
\emph{lipskyana} does not currently occur sympatrically with
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} (time 4;
Fig~\ref{fig:6}b). This suggests that one or both of these taxa
underwent a recent range contraction since their hybridization.

An additional gene flow event may have occurred from \emph{P.~rex}
subsp.\ \emph{rex} into either both subspecies of
\emph{P.~thamnophila}, or perhaps into only \emph{P.~thamnophila}
subsp.\ \emph{cupuliformes}, at some point after time 3. This would
seem the most parsimonious explanation for the presence of alleles
derived in \emph{P.~rex} subsp. \emph{rex} relative to \emph{P.~rex}
subsp. \emph{rockii} being present in both subspecies of
\emph{P.~thamnophila}, yielding a significant D$_{12}$,
while neither subspecies of \emph{P.~thamnophila} appears to have
uniquely derived alleles, relative to each other, introgressed into
\emph{P.~rex} subsp.\ \emph{rex}.  In interpreting this result we are
limited by having only two samples from \emph{P.~thamnophila}. Because
D-statistics are a comparative measurement, showing only whether gene
flow occurred into one taxon more so than into another, it is
difficult to determine whether the taxon which received less gene flow
in fact received any at all, unless there is another sample which
received even less gene flow with which to compare it.

Our clearest result, which holds independently of whether the two
subspecies of \emph{P.~thamnophila} are monophyletic or paraphyletic,
shows that \emph{P.~thamnophila} subsp.\ \emph{thamnophila} exchanged
genes with \emph{P.~rex}. If \emph{P.~thamnophila} is monophyletic,
our results can additionally be interpreted to show a pattern of
highly asymmetric gene flow, with introgression occurring from
\emph{P.~rex} much more than in the reverse direction. Such asymmetry
is not unexpected in a hybridizing pair when one species is
comparatively rare \citep{levin_hybridization_1996}, as introgressed
alleles could spread quickly through a small population, whereas they
are more likely to remain localized in a more widespread species.
This could be the case among the widespread and common \emph{P.~rex}
taxa as they hybridized with the narrow endemic \emph{P.~thamnophila}
taxa. %subsp.\ \emph{thamnophila}.
Introgression in this way can even pose a potential extinction risk to
the rarer species \citep{ghosh_quantifying_2012}, particularly if gene
flow has only recently been initiated due to a range expansion.  With
the methods and data used here, it is difficult to ascertain whether
introgression occurred persistently through time versus having
occurred only recently; however, our results suggest gene flow into
\emph{P.~thamnophila} subsp.\ \emph{thamnophila} occurred from several
distinct lineages of \emph{P.~rex}, including from one which is now
geographically isolated and thus likely to have occurred in the past,
and two events which are very recent. With these hypotheses in mind,
more explicit model-based tests could be used to more accurately infer
the timing of divergence and gene flow.  This includes
isolation-migration models \citep[e.g., IMa2;][]{hey_isolation_2010},
or more complex simulation-based models implemented in an approximate
Bayesian computation framework \citep{beaumont_approximate_2010}.


\subsection{Floral divergence and isolation}
In \emph{Pedicularis} sect.\ \emph{Cyathophora}, the most conspicuous
morphological differences among species are in their flowers, with the
majority of variation composing different combinations of three floral
characters. These include the length of the corolla tube, which
covaries with the presence versus absence of nectar as well as the
length that pollen tubes must grow to fertilize the ovules; the length
and curvature of the galea (fused upper lobes of corolla) which
directs pollen placement onto either the dorsal or ventral side of
visiting pollinators; and flower color, which is typically yellow,
reddish-purple, white, or some combination thereof
(Fig.~\ref{fig:1}). All species in the superba clade have
reddish-purple flowers, but exhibit a wide variety of floral
morphologies, one species having a long corolla tube and long-beaked
galea (\emph{P.~cyathophylla}), one a medium-length corolla tube and
short, slightly beaked galea (\emph{P.~cyathophylloides}), and the
other a short corolla tube and short-beaked galea (\emph{P.~superba}).
Taxa in the rex-thamnophila clade, by contrast, all have short corolla
tubes and short, rounded, beakless galeas.  The flowers of
\emph{P.~thamnophila} are smaller, have a spreading lower corolla lip,
and are more consistently yellow, whereas those of \emph{P.~rex} are
larger, have an adpressed lower lip, and vary in flower color, with
yellow, whitish, and reddish-purple forms (the latter characterizing
\emph{P.~rex} subsp.\ \emph{lipskyana}).

Floral differences do not seem to closely reflect phylogenetic
distances, as taxa in the superba clade exhibit much more
differentiated flowers than those in the rex-thamnophila clade.  A
potential explanation for the great diversity of flowers in
\emph{Pedicularis} is reproductive character displacement
\citep{armbruster_floral_1994}, particularly, in the way it may occur
among the many closely related species which co-occur, flower
synchronously, and share pollinators in the Hengduan region.
\cite{eaton_floral_2012} found support for this hypothesis at the
community scale, showing both consistent overdispersion (i.e.,
limiting similarity) of floral traits among co-occurring
\emph{Pedicularis} species across local communities, as well as a
phylogenetic signal of homoplasy in the evolution of floral traits,
suggesting there has been persistent selection driving repeated
adaptations to fill available floral niches.  A more specific
hypothesis, however, is that such selection, rather than being driven
by all species of \emph{Pedicularis} that locally co-occur, may
instead be caused primarily by interactions among only the most
closely related species -- those still capable of exchanging genes.
\cite{eaton_floral_2012} found some support for this hypothesis,
showing that only the most species-rich communities tend to compose
more distantly related species than expected.

This study, in showing different levels of gene flow among taxa in
clades exhibiting different degrees of floral differentiation, offers
insight into the processes of floral divergence. If differences are
driven primarily by selection to reduce interspecific gene flow -- the
process of reinforcement \citep{hopkins_pollinator-mediated_2012,
  servedio_role_2003} -- then species that experienced past
introgression are expected to exhibit more differentiated floral
morphologies today.
Such a pattern is the opposite of what we observe in
\emph{Pedicularis} sect.\ \emph{Cyathophora}, namely greater gene flow
among taxa with more similar flowers.  Our results are consistent with
a reproductive character displacement scenario in which introgression
between taxa inhibits morphological differentiation.  This is further
supported by the fact that taxa in the superba clade tend to occur at
higher elevations and in smaller, more isolated populations, compared
to taxa in the rex-thamnophila clade.  Following this line of
reasoning, floral divergence does not occur during speciation, but
rather, taxa which become geographically isolated from close relatives
experience greater opportunity for adaptation to local conditions,
which in turn sets the stage for further evolution of reproductive
isolation.

Several recent studies have shown a contrasting pattern, in which
hybridization appears to have played a role in generating phenotypic
diversity.  In \emph{Heliconius} butterflies, for example,
introgression between closely related species has been shown
to allow the exchange of supergenes underlying similar mimicry patterns
\citep{the_heliconius_genome_consortium_butterfly_2012}; or in
Louisiana irises, where hybrids show novel phenotypes with increased
fitness relative to their parent species \citep{arnold_hybrid_2012}.
In \emph{Pedicularis}, if hybrid introgression has contributed to
adaptive radiation by similarly facilitating the exchange of genes
affecting floral phenotypes, it could help explain the high degree of
floral diversity and homoplasy in the group
\citep{ree_phylogeny_2005}; however, no evidence for this has yet been
found.

Central to this explanation is the rate at which reproductive
isolating barriers evolve, and the nature of these barriers in
\emph{Pedicularis}. Pre-mating isolation facilitated by the
differentiation of floral traits can play a significant role in the
speciation process, particularly in a clade such as the one studied
here, where close relatives capable of hybridization occur
sympatrically. In rex-thamnophila, although species exhibit less
differentiation in floral traits than in the superba clade, they do
exhibit small differences that could have large effect. Pre- and
post-zygotic barriers are needed to complete the cessation of gene
flow, and in \emph{Pedicularis}, crossing experiments need to be done
to provide empirical evidence of current hybridization potential with
genomic estimates of past gene flow.

Estimates of hybridization based on morphological determinations
indicate that it occurs in as many as 25\% of plant species and 10\%
of animal species \citep{mallet_hybridization_2005}, but the
phenomenon remains rarely investigated and poorly understood in a
phylogenetic context. Ideally, phylogenies that accurately represent
the history of population divergences could be applied to study
patterns of introgression in a comparative framework, where its
effects on character evolution and rates of divergence could be
inferred. Because interspecific gene flow confounds phylogeny
reconstruction, however, such studies remain difficult.  In
\emph{Pedicularis} sect. \emph{Cyathophora}, introgression was found
to significantly affect phylogenetic inference.  By applying
genome-wide tests for introgression to RADseq data, we were able to
identify hybridizing taxa, and to compare specific phylogenetic
hypotheses after minimizing the effect of introgressed DNA.  This
allowed us to recover a new topology not originally supported by any
of the phylogenetic inference methods employed, and which was more
consistent with geography and morphology.  Future analyses,
particularly in plants, are likely to benefit from integrating genomic
information about hybrid introgression into phylogenetic analyses.

\section{Supplemental Material}
Supplementary material, including data files and/or online-only appendices, 
can be found in the Dryad data repository (doi:10.5061/dryad.bn281).

\section{Funding}
Funding for this research was provided by National Science Foundation grants
(DEB-1119098) to R.H.R.\ and (DEB-1110598) to D.A.R.E.

\section{Acknowledgments}
We thank B.~Rubin, P.~Grabowski, J.~Borevitz, R.~Edgar and D.~Baum
for insightful discussions on the analysis of RAD data,
Floragenex Inc. for RADseq preparation, and 
C. An\'e, R. Olmstead and one anonymous reviewer for comments that 
improved the manuscript.


%%% bibtex .bbl data
\bibliography{refs}
\bibliographystyle{ecol_let}


\section{Figures and Tables}
\subsection{Tables}

\begin{sidewaystable}
%\begin{table}
  \caption{Results of filtering and clustering RAD sequences from 13
    individuals of \emph{Pedicularis}, identified in subsequent tables
    by codes in the ID column. The number of loci (clusters) having
    each sample in the minimum-taxa and full-taxa data sets are
    shown.}
\label{tab:1}
\begin{center}
\begin{tabular*}{1.0\textwidth}{@{\extracolsep{\fill}}llccccccc}
\hline
\multirow{2}{*}{Taxon}   &\multirow{2}{*}{ID}   & RAD tags       & Clusters   &  Mean     & Consensus & Minimum-taxa & Full-taxa \\
                         &                      & (\e{6})        &  at 90\%\ensuremath{^{a}}   & depth   & loci\ensuremath{^{b}}  & data set & data set  \\ 
\hline
\hline
\emph{P.~rex}~subsp.\ \emph{rex}                   &R$_{r1}$   &1.71   & 54~832   &24.45   &51~525  &35~021   &  4~869 \\  %40578
\emph{P.~rex}~subsp.\ \emph{rex}                   &R$_{r2}$   &1.41   & 54~220   &20.11   &49~556  &33~991   &  4~869 \\  %35855
\emph{P.~rex}~subsp.\ \emph{lipskyana}             &R$_{l1}$   &1.39   & 51~754   &22.53   &48~962  &34~873   &  4~869 \\  %38362
\emph{P.~rex}~subsp.\ \emph{lipskyana}             &R$_{l2}$   &0.82   & 41~576   &13.95   &38~653  &28~351   &  4~869 \\  %39618
\emph{P.~rex}~subsp.\ \emph{rockii}                &R$_{o}$    &1.80   & 53~135   &21.68   &50~020  &34~313   &  4~869\\   %33405 -- 35236
\emph{P.~thamnophila}~subsp.\ \emph{thamnophila}   &T$_t$      &1.45   & 51~146   &21.57   &47~052  &32~791   &  4~869 \\  %30556 is T2
\emph{P.~thamnophila}~subsp.\ \emph{cupuliformis}  &T$_c$      &0.64   & 27~555   &12.13   &25~215  &18~054   &  4~869 \\  %33413 is T1
\emph{P.~cyathophylloides}                        &C$_1$      &2.20   & 53~959   &34.61   &51~258  &31~559   &  4~869 \\  %41478
\emph{P.~cyathophylloides}                        &C$_2$      &2.20   & 73~880   &22.59   &69~517  &29~297   &  4~869 \\  %41954
\emph{P.~cyathophylla}                            &Y         &1.25    & 50~357   &16.37   &46~510  &26~053   &  4~869 \\  %30686
\emph{P.~superba}                                 &S         &0.70    & 32~970   &14.16    &30~628  &20~726   &  4~869 \\  %29154
\emph{P.~przewalskii}                             &W$_1$     &0.96    & 39~621   &17.00    &36~231  &12~244   &  2~631 \\  %32082
\emph{P.~przewalskii}                             &W$_2$      &1.00   & 44~207   &16.25    &40~670  &14~288   &  2~993 \\  %33588
\hline
\hline
%$Mean$                                          &1.35 &1.19  & 206~866   &18.01   &0.73  &0.37  &  49~077\\
%$SD$                                            &0.52 &0.47  & ~75~145   &~5.37   &0.21  &0.12  &  13~274 \\
\end{tabular*}
%\begin{tabular*}{1.0\textwidth}
\bigskip
\end{center}
\noindent $^{a}$ clusters with more than the minimum depth of five reads.\\
\noindent $^{b}$ consensus loci which passed filtering for paralogs.
\end{sidewaystable}

\clearpage
\newpage


\begin{table}
  \caption{Patterson's four-taxon D-statistic measure of introgression presented as a Z-score, applied to RAD sequences simulated under the ``weak gene flow'' scenario, where unidirectional introgression occurred from taxon C$_i$ into B$_i$ as shown in Figure 2; some tests do not include both taxa. In each test, taxa are arranged such that the ABBA pattern is more frequent than BABA. Significant results are in bold.}
% Tests 1.1 and 1.2 accurately detected introgression among taxa which hybridized in the simulations.
% Tests 2.1-2.10 did not detect introgression, as expected, given the taxa involved did not hybridize directly. Tests 3.1-3.10 also involve taxa which did not hybridize directly, but demonstrate examples 
% where significant D-statistics are still detected. 
\label{tab:2}
\begin{center}
\begin{tabular*}{0.5\textwidth}{@{\extracolsep{\fill}}lccccccc}
\hline
Test         & P1    & P2       & P3      & O       &Z    \\
\hline
\hline
1.1          & A    & B$_i$    & C$_i$    & o       &\bf11.34  \\
1.2          & B    & B$_i$    & C$_i$    & o       &\bf14.47  \\
%\hline
2.1          & A    & B        & C$_i$    & o       &0.26  \\
2.2          & A    & B        & C        & o       &0.03  \\
2.3          & A    & B        & D        & o       &0.56  \\
2.4          & C    & D        & A        & o       &1.66  \\
2.5          & C    & D        & B        & o       &1.01  \\
2.6          & C    & C$_i$    & A        & o       &0.58  \\
2.7          & C    & C$_i$    & B        & o       &0.36  \\
2.8          & C    & C$_i$    & D        & o       &0.44  \\
2.9          & C    & C$_i$    & D        & A       &0.50  \\
2.10         & C    & C$_i$    & D        & B       &0.20  \\
%\hline
3.1          & B$_i$  & B        & A        & o       &\bf7.30 \\
3.2          & B$_i$  & B        & A        & D       &\bf13.73\\
3.3          & B$_i$  & B        & A        & C       &\bf15.43\\
3.4          & B      & B$_i$    & C        & o       &\bf15.88\\
3.5          & A      & B$_i$    & C        & o       &\bf9.96\\
3.6          & B      & B$_i$    & D        & o       &\bf8.74\\
3.7          & A      & B$_i$    & D        & o       &\bf5.48\\
3.8          & D      & C        & B$_i$    & o       &\bf4.38\\
3.9          & D      & C$_i$    & B$_i$    & o       &3.08\\
3.10         & C      & C$_i$    & B$_i$    & o       &\bf5.87\\
\hline
\hline
\end{tabular*}
\end{center}
\end{table}

\clearpage
\newpage

\begin{table}
  \caption{Partitioned D-statistic test for introgression performed on the ``weak gene flow'' and ``multiple-reticulation'' simulated data sets. Results are presented as Z-scores for D$_{12}$, D$_1$, and D$_2$, respectively. As in Table 2, some tests do not include all taxa participating in introgression. In each row, taxa are arranged such that the dominant pattern indicates introgression into P2 (i.e., A~B~\_~\_~A). Significant results are in bold.}
\label{tab:3}
\begin{center}
\begin{tabular*}{0.80\textwidth}{@{\extracolsep{\fill}}lccccc|ccc|ccc}
\hline
\multicolumn{6}{}{}  & \multicolumn{3}{c}{C$_i$ into B$_i$}  & \multicolumn{3}{c}{C and C$_i$ into B$_i$} \\
\hline
Test         & P1       & P2       & P3$_1$       & P3$_2$       & O  & Z$_{12}$  & Z$_1$ & Z$_2$  & Z$_{12}$  & Z$_1$ & Z$_2$  \\
\hline
\hline
4.1          &B        & B$_i$    & C           & D      & o     & \bf8.71  &   \bf12.62  &  1.40      & \bf18.27  &   \bf30.90 &  1.66    \\
4.2          &A        & B$_i$    & C           & D      & o     & \bf6.28  &   \bf9.31   &   1.37    & \bf15.39  &   \bf22.30   & 1.96     \\
4.3          &B         &B$_i$    & C$_i$        &C      &o      & \bf12.39   &    \bf7.66   &  0.68  & \bf23.97   &  \bf11.10   &  \bf10.31 \\
4.4          &B         &B$_i$    & C$_i$        &D      &o      & \bf9.35    &    \bf13.89  &  0.71  & \bf18.07   &  \bf33.90   &  1.63    \\
4.5          &A         &B$_i$    & C$_i$        &C      &o      & \bf10.17   &    \bf6.64   &  0.32  & \bf22.68   &  \bf5.11    &  \bf6.56 \\
4.6          &A         &B$_i$    & C$_i$        &D      &o      & \bf6.53    &    \bf11.74  &  1.61  & \bf15.41   &  \bf21.82   &  2.13    \\
5.1          &A         &B        &C             &D      &o     & 0.32      &     0.52     &  2.04   & 2.56       &    0.84     &  0.97    \\
5.2          &A         &B        &C$_i$         &C      &o     &0.27       &     0.11     &  0.93   & 2.89       &    0.92     &  0.32     \\
5.3          &A         &B        &C$_i$         &D      &o    &0.17       &     0.26     &  1.86    & 2.43       &    0.83     &  1.24     \\
5.4          &C         &D        &B             &A      &o    &1.47       &     0.47     &  1.06    & 1.26       &    0.54     &  1.23     \\
5.5          &D         &C$_i$    &B             &A      &o    &2.07        &    0.79     &   1.31   & 0.50       &    0.12     &  1.96     \\
6.1          &C         &C$_i$    &B$_i$         &B      &o     &0.28       &   \bf 7.84   &  0.53   & 2.50       &    1.86     &  1.28     \\
6.2          &C         &C$_i$    &B$_i$         &A      &o     &0.20       &   \bf 6.40   &  0.45   & 2.50       &    1.85     &  0.21     \\
6.3          &D         &C$_i$    &B$_i$         &A      &o     &1.40       &   \bf 11.14  &  1.68   & 0.28       &    \bf21.09 &  2.59     \\
6.4          &D         &C$_i$    &B$_i$         &B      &o     &1.23       &   \bf 14.01  &  0.70   & 0.46       &    \bf30.40 &  2.06     \\
6.5          &D         &C        &B$_i$         &A      &o     &1.15       &   \bf 9.87   &  1.27   & 1.91       &    \bf20.90 &  1.90    \\
6.6          &D         &C        &B$_i$         &B      &o     &1.01       &   \bf 11.70  &  0.77   & 1.99       &    \bf32.25 &  1.08    \\
7.1          &B         &B$_i$    &A             &C$_i$  &o     &0.29       &   \bf 9.68   &\bf16.06  &0.58        &   \bf 21.22 &  \bf35.54 \\ 
7.2          &B         &B$_i$    &A             &C      &o     &0.57     &  \bf 9.68    & \bf16.31  &0.63        &  \bf 21.79  &  \bf33.54 \\ 
7.3          &C         &C$_i$    &D             &B$_i$  &o    &0.91      & 1.35         & \bf3.65   & 2.01       &  0.74       & 2.21      \\
7.4          &C         &C$_i$    &D             &B      &o    &0.27      & 1.23        & 1.22       & 2.17       &  0.47       & 1.85      \\
\hline
\hline
\end{tabular*}
\end{center}
\end{table}
\clearpage
\newpage


\begin{table}
\caption{Patterson's four-taxon D-statistic test for introgression in \emph{Pedicularis} sect.\ \emph{Cyathophora}. Each test was repeated over all possible four-sample replicates (n), with a range of Z-scores reported, and the number of significant replicates shown (nSig). Taxa are identified by codes listed in Table 1, with numeric subscripts distinguishing individual samples. When no subscript is given, test replicates include all individuals sampled (e.g., R$_r$ = R$_{r1}$ or R$_{r2}$; CS = C$_1$ or C$_2$ or S). In each row, taxa are arranged such that the dominant pattern is always ABBA.}

\label{tab:4}
\begin{center}
\begin{tabular*}{0.75\textwidth}{@{\extracolsep{\fill}}lllllcc}
\hline
Test         & P1   & P2       & P3          & O           &Range Z      & nSig / n \\
\hline
\hline
8.1          & C$_1$  & C$_2$  & RT         & W            &N/A             & 0/14 \\
8.2          & Y    & CS      & RT         & W            &(0.00, 3.24)    & 0/42 \\
8.3          & S    & CY      & RT         & W            &(0.92, 2.77)    & 0/42 \\
8.4          & RT  & RT      & C           & W            &(0.06, 2.81)    & 0/84 \\
8.5          & RT  & RT      & Y           & W            &(0.06, 2.60)    & 0/42 \\
8.6          & RT  & RT      & S           & W            &(0.00, 1.88)    & 0/42 \\
9.1          & S    & Y        & C         & RT           &(1.35, 3.03)    & 0/14 \\
9.2          & C$_1$& C$_2$    & Y         & RT           &N/A              & 0/7    \\
9.3          & C$_1$& C$_2$    & S         & RT           &N/A              & 0/7      \\
10.1         & R    & T$_t$    & T$_c$     & CYS          &(5.05, 11.14)   & {\bf20}/20 \\
%10.12          & R$_{r1}$ & R$_{r2}$ & T$_c$    & C+Y+S        &(0.35, 1.48)    & 0/4    \\
%10.11          & R$_{l1}$ & R$_{l2}$ & T$_c$    & C+Y+S        &(0.00, 0.75)    & 0/4      \\
10.2          & R$_{l}$ & R$_{r}$ & T$_c$      & CYS        &(0.00, 1.73)    & 0/16     \\
10.3          & R$_{o}$ & R$_{r}$ & T$_c$      & CYS        &(2.84, 5.06)    & {\bf8}/8 \\
10.4          & R$_{o}$ & R$_{l}$ & T$_c$      & CYS        &(1.43, 3.04)    & 0/8       \\
%10.13         & R$_{r1}$ & R$_{r2}$ & T$_t$    & C+Y+S        &(0.54, 1.94)    & 0/4        \\
%10.14         & R$_{l1}$ & R$_{l2}$ & T$_t$    & C+Y+S        &(0.06, 0.97)    & 0/4        \\
10.5          & R$_{l}$ & R$_{r}$ & T$_t$     & CYS        &(1.52, 2.96)    & 0/16      \\
10.6         & R$_{o}$ & R$_{r}$ & T$_t$      & CYS        &(7.15, 8.65)    & {\bf8}/8  \\
10.7         & R$_{o}$ & R$_{l}$ & T$_t$      & CYS        &(4.26, 6.20)    & {\bf8}/8  \\
10.8         & T$_c$   & T$_t$  & R$_o$      & CYS        &(8.96, 11.20)   &  {\bf4}/4 \\
10.9         & T$_c$   & T$_t$  & R$_r$      & CYS        &(9.48, 12.07)   &  {\bf4}/4 \\
10.10        & T$_c$   & T$_t$  & R$_l$      & CYS        &(8.92, 11.79)   &  {\bf4}/4 \\
%11.1         & R$_o$   & R$_l$  & R$_r$      & CYS        &(0.10, 2.48)    & 0/16      \\
%11.2         & R$_o$   & R$_r$  & R$_l$      & CYS        &(1.48, 3.66)    & ?/16      \\
%11.3         & R$_l$   & R$_r$  & R$_o$      & CYS        &(0.62, 2.29)    & 0/16      \\

\hline
\hline
\end{tabular*}
\end{center}
\end{table}
\clearpage
\newpage


\begin{sidewaystable}
\caption{Partitioned D-statistic test for introgression in \emph{Pedicularis} sect.\ \emph{Cyathophora}. Each test was repeated over all possible five-taxon subtree replicates (n) to yield a range of Z-scores. The number of significant replicates is given by nSig. Taxa are identified using codes following Table 4. Z-scores are reported for each respective D-statistic, representing asymmetry in incongruent allele patterns for which the derived allele is shared by both P3$_1$ and P3$_2$ (Z$_{12}$), by P3$_1$ but not P3$_2$ (Z$_1$), or by P3$_2$ but not P3$_1$ (Z$_2$). Tests are arranged such that the dominant pattern is always introgression into P2 (i.e., A~B~\_~\_~A).}
\label{tab:5}
\begin{center}
\begin{tabular*}{0.97\textwidth}{@{\extracolsep{\fill}}lccccc|cc|cc|cc}
\hline
\multicolumn{6}{}{} & \multicolumn{2}{c}{P3$_1$ \& P3$_2$} & \multicolumn{2}{c}{P3$_1$ only}  & \multicolumn{2}{c}{P3$_2$ only} \\
\hline
Test    & P1       & P2       & P3$_1$      & P3$_2$        & O       & Z$_{12}$         & nSig/n   & Z$_1$  & nSig/n   & Z$_2$  &  Nsig/n  \\ 
\hline
\hline
11.1     & R$_o$    & R$_r$    & T$_t$     & T$_c$   & CYS    &(3.31, 5.80)   & {\bf8}/8    & (1.12, 3.71)   & {\bf1}/8 & (0.09, 2.29)    & 0/8  \\
11.2     & R$_o$    & R$_l$    & T$_t$     & T$_c$   & CYS    &(1.52, 2.78)   & 0/8         & (2.19, 3.74)   & {\bf2}/8 & (0.33, 2.15)    & 0/8  \\
11.3     & R$_r$    & R$_l$    & T$_t$     & T$_c$   & CYS    &(0.00, 2.44)   & 0/16        & (0.07, 1.29)   & 0/16     & (0.24, 1.98)    & 0/16  \\
%11.4     & R$_r$    & R$_l$    & T$_t$     & T$_c$   & C+Y+S    &(0.63, 1.33)   & 0/16        & (0.25, 0.74)   & 0/16     & (0.69, 1.99)    & 0/16 \\

12.1     & T$_c$    & T$_t$    & R$_o$     & R$_{r}$     &CYS    &(7.54, 10.90)   &{\bf8}/8    & (3.31, 5.65)   & {\bf8}/8  & (5.00, 7.51)  & {\bf8}/8  \\
12.2     & T$_c$    & T$_t$    & R$_o$     & R$_{l}$     &CYS    &(6.91, 10.99)   &{\bf8}/8    & (2.56, 5.24)   & {\bf7}/8  & (4.22, 6.77)  & {\bf8}/8  \\
12.3     & T$_c$    & T$_t$    & R$_{r}$   & R$_{l}$     &CYS    &(8.03, 10.99)   & {\bf16}/16 & (4.64, 7.77)   & {\bf16}/16  & (3.45, 6.84) & {\bf16}/16 \\
12.4     & T$_c$    & T$_t$    & R$_{r1}$   & R$_{r2}$    &CYS    &(10.60, 11.75) & {\bf4}/4   & (1.88, 2.83)   & 0/4       & (3.17, 4.05)   & {\bf4}/4  \\
12.5     & T$_c$    & T$_t$    & R$_{l1}$   & R$_{l2}$    &CYS    &(7.93, 11.15)  & {\bf4}/4   & (0.44, 1.18)   & 0/4       & (3.25, 5.44)   & {\bf4}/4  \\

13.1     & R$_{r2}$  & R$_{r1}$  & R$_l$    & T$_t$      &CYS    &(0.18, 0.85)    & 0/8        & (0.27, 2.73) & 0/8         & (0.19, 1.28)   & 0/8   \\
13.2     & R$_{l2}$  & R$_{l1}$  & R$_r$    & T$_t$      &CYS    &(0.18, 1.19)    & 0/8        & (0.23, 1.57) & 0/8         & (0.20, 1.24)   & 0/8   \\
13.3     & R$_{r2}$  & R$_{r1}$  & R$_o$    & T$_t$      &CYS    &(0.41, 1.13)    & 0/4        & (0.07, 1.37) & 0/4         & (1.15, 2.30)   & 0/4   \\
13.4     & R$_{l2}$  & R$_{l1}$  & R$_o$    & T$_t$      &CYS    &(0.30, 0.94)    & 0/4        & (0.25, 0.85) & 0/4         & (1.17, 1.89)   & 0/4   \\

\hline
\hline
\end{tabular*}
\end{center}
\end{sidewaystable}
\clearpage
\newpage


\subsection{Figure captions}


\noindent Figure~\ref{fig:1}: Sampling localities and photographs of taxa in \emph{Pedicularis} sect.~\emph{Cyathophora}. Codes for taxon names are listed in Table~\ref{tab:1}. \\

\noindent Figure~\ref{fig:2}: Coalescent model used to simulate RADseq-like data. Unidirectional migration (gene flow) from taxon C$_i$ into B$_i$ (solid gray arrow) occurred over 10,000 generations, beginning 50,000 generations in the past. Three data sets were created with different rates of gene flow, and one additional data set with gene flow from both C and C$_i$ into B$_i$ (dashed gray arrow). \\

\noindent Figure~\ref{fig:3}: A schematic description of Patterson's four-taxon D-statistic test for hybridization and the five-taxon partitioned D-statistic test. (a) The four-taxon test measures asymmetry in the occurrence of two incongruent allele patterns (shown in the box), which are expected to arise with equal frequencies in the absence of introgression, but to deviate from symmetry if P3 exchanged genes with either P1 or P2 to the exclusion of the other. The five-taxon test samples an additional population within the P3 lineage, the two samples now denoted P3$_1$ and P3$_2$, and measures asymmetry in three sets of allele patterns (the three boxes shown), thus measuring three separate D-statistics. The first (D$_1$) measures asymmetry in incongruent alleles where the derived allele is present in P3$_1$ but not P3$_2$, the second (D$_2$) where it is present in P3$_2$ but not P3$_1$, and the third where the derived allele is shared by both P3$_1$ and P3$_2$, having arisen in their ancestor (branch indicated by **). \\

\noindent Figure~\ref{fig:4}: Phylogeny of \emph{Pedicularis} sect. \emph{Cyathophora} inferred from RADseq data. ML trees were estimated from the sparse (minimum-taxa) supermatrix (a) and the densely sampled (full-taxa) supermatrix (b), yielding high bootstrap support (100 except where indicated). Primary concordance trees (c,d) and population trees (e,f) were inferred on 945 variable loci from the full-taxa supermatrix, at either $\alpha$ = 0.01 (c, e) or 100 (d, f). The 95\% CI for CFs are shown on primary concordance trees, those in bold did not overlap with any conflicting CF. Quartet CFs are shown on population trees. \\

\noindent Figure~\ref{fig:5}. Censored data sets, in which one or more taxa have been removed to minimize the effect of their introgressed alleles, are used to compare alternative topologies within the Shimodeira-Hasegawa test. The ML topology with \emph{P.~rex} subsp.\ \emph{rex} and \emph{P.~rex} subsp.\ \emph{lipskyana} removed (a) is compared with an alternative topology in which \emph{P.~thamnophila} is monophyletic (b). Similarly, the ML topology with both subspecies of \emph{P.~thamnophila} removed (c) is compared with two alternative resolutions for the three subspecies of \emph{P.~rex} (d \& e). \\

\noindent Figure~\ref{fig:6}: Temporal and geographic reconstructions of population divergence and gene flow, as inferred from the partitioned D-statistic. Codes for taxon names are listed in Table~\ref{tab:1}. (a) Phylogeny of rex-thamnophila based on censored subtree comparisons which account for hybrid introgression. Grey bars indicate distinct time periods discussed in the text, and arrows are the direction of introgressive gene flow. (b) Schematic reconstruction of historical biogeography in the rex-thamnophila clade, inferred from present-day distributions and patterns of historical introgression. Overlapping edges indicate geographic overlap, and arrows represent introgression. \\


\clearpage
\newpage

\begin{figure}
\begin{center}
%\includegraphics[width=.75\textwidth]{final_figs/Fig1}
\end{center}
\caption{}
\label{fig:1}
\end{figure}

\begin{figure}
\begin{center}
%\includegraphics[width=.50\textwidth]{final_figs/Fig2_simsetup}
\end{center}
\caption{}
\label{fig:2}
\end{figure}

\begin{figure}
\begin{center}
%\includegraphics[width=.9\textwidth]{final_figs/Fig3_revised_partD}
\end{center}
\caption{}
\label{fig:3}
\end{figure} 

\begin{figure}
\begin{center}
%\includegraphics[width=.90\textwidth]{final_figs/Fig4}
\end{center}
\caption{}
\label{fig:4}
\end{figure}

\begin{figure}
\begin{center}
%\includegraphics[width=.90\textwidth]{final_figs/Fig5_revised_SHtest}
\end{center}
\caption{}
\label{fig:5}
\end{figure}

\begin{figure}
\begin{center}
%\includegraphics[width=.75\textwidth]{final_figs/Fig6_revised_tempgeo}
\end{center}
\caption{}
\label{fig:6}
\end{figure}

\end{document}


%%% Local Variables: 
%%% mode: tex-pdf
%%% TeX-master: t
%%% End: 

